src/main/java/frc/robot/utils/ThreadedGyro.java

```diff
@@ -0,0 +1,59 @@
+package frc.robot.utils;
+
+import com.kauailabs.navx.frc.AHRS;
+
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+public class ThreadedGyro {
+    private final AHRS gyro;
+    private final AtomicBoolean shouldReset = new AtomicBoolean(false);
+    private volatile double lastGyro;
+    private volatile double angleAdjustment;
+    private boolean enabled = false;
+    private final ScheduledExecutorService executor;
+
+    public ThreadedGyro(AHRS gyro) {
+        this.gyro = gyro;
+        this.angleAdjustment = gyro.getAngleAdjustment();
+        this.executor = Executors.newScheduledThreadPool(1);
+    }
+
+    public void execute(){
```

Comment:
This method should probably be named `start()`

Reply:
done